flink-table/flink-sql-parser/src/test/java/org/apache/flink/sql/parser/FlinkSqlParserImplTest.java

```diff
@@ -95,6 +95,21 @@ public void testUseCatalog() {
 		check("use catalog a", "USE CATALOG `A`");
 	}
 
+	@Test
+	public void testCreateCatalog() {
+		check(
+			"create catalog c1\n" +
+				" WITH (\n" +
```

Comment:
We can test with compound identifier directly, e.g. 'a.b.c1', because that is actually what the syntax supports.

Reply:
it is catalog, not table, so it doesn't have name space

Reply:
But your syntax allows that.